code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks_vr.dm

```diff
@@ -106,4 +106,300 @@
 	..()
 	playsound(M, 'sound/items/hooh.ogg', 50, 1, -1)
 	if(prob(5))
-		to_chat(M, "<span class='warning'>You feel like you've been gnomed...</span>")
\ No newline at end of file
+		to_chat(M, "<span class='warning'>You feel like you've been gnomed...</span>")
+
+
+/datum/reagent/ethanol/galacticpanic
+	name = "Galactic Panic Attack"
+	id = "galacticpanic"
+	description = "The absolute worst thing you could ever put in your body."
+	taste_description = "an entire galaxy collasping in on itself."
+	strength = 10
+	druggy = 50
+	halluci = 30
+	var/adj_dizzy = 10
+	color = "#d3785d"
+
+	glass_name = "Galactic Panic Attack"
+	glass_desc = "Looking into this is like staring at the stars."
+
+/datum/reagent/ethanol/galacticpanic/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
+	..()
+	M.Stun(2)
+
+/datum/reagent/ethanol/galacticpanic/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
+	..()
+
+	if(dose * strength >= strength) // Early warning
+		M.make_dizzy(24) // Intentionally higher than normal to compensate for it's previous effects.
+	if(dose * strength >= strength * 2.5) // Slurring takes longer. Again, intentional.
+		M.slurring = max(M.slurring, 30)
+
+/datum/reagent/ethanol/bulldog
+	name = "Space Bulldog"
+	id = "bulldog"
+	description = "An inventive kahlua recipe."
+	taste_description = "fizzy, creamy, soda and coffee hell."
+	strength = 30
+	color = "#d3785d"
+
+	glass_name = "Space Bulldog"
+	glass_desc = "It looks like someone poured cola in a cup of coffee."
+
+
+/datum/reagent/ethanol/sbagliato
+	name = "Negroni Sbagliato"
+	id = "sbagliato"
+	description = "A drink invented because a bartender was too drunk."
+	taste_description = "sweet bubbly wine and vermouth."
+	strength = 30
+	color = "#d3785d"
+
+	glass_name = "Negroni Sbagliato"
+	glass_desc = "Bubbles constantly pop up to the surface with a quiet fizz."
+
+
+/datum/reagent/ethanol/italiancrisis
+	name = "Italian Crisis"
+	id = "italiancrisis"
+	description = "This drink was concocted by a madwoman to cause to the Italian Crisis of 2123."
```

Comment:
```suggestion
	description = "This drink was concocted by a madwoman because of the Italian Crisis of 2123."
```

The original description read; "This drink was concocted by a madwoman to cause to the Italian Crisis of 2123."

I don't understand 'to cause to the Italian Crisis', it doesn't flow very well.

Reply:
Pip mashing her hands again. Committed.

Reply:
GAH! I fat fingered it. I mean to write "This drink was concocted by a madwoman to cause the Italian Crisis of 2123."